python/dgl/nn/pytorch/explain/pgexplainer.py

```diff
@@ -361,7 +370,7 @@ def explain_graph(self, graph, feat, tmp=1.0, training=False, **kwargs):
         ...     optimizer.step()
 
         >>> # Initialize the explainer
-        >>> explainer = PGExplainer(model, data.gclasses)
+        >>> explainer = PGExplainer(model, data.gclasses, num_hops=1)
```

Comment:
Is this argument required for `explain_graph`? If not, maybe make this argument optional and set its default value to `None`.

Reply:
it's not required, I'll follow this and also add an assertion for `num_nodes` provided during initialization